### ISSUE TITLE
Exposed server issue.

### ISSUE THREAD

**Issue by DaemonPhoenix42** [happypandax/happypandax#67]
Not sure how much of an issue this is but the server didn't expose with the value of "localhost", but I got my phone to connect but specifying my servers local IP.
Not sure why that is.

**Comment by emmauss** [comment]
what interface does the console say it is listening on? You can change it by modifying the hos and web addresses in config

**Comment by DaemonPhoenix42** [comment]
I'm new to wrap my head around this software but I'm not sure what you mean by interface.
Through HPX_GUI the web_client address was blank by default and I tried localhost.
In both cases, I had no issue connecting on the host machine to the web_client but was unable to access it from another network device i.e. Android/laptop. and yes "expose_server" was enabled.
Now after specifying my host machine local IP "192.168.2.86" I was then able to connect from my other device with no issue.
My problem is, wouldn't specifying the address be synonymous with "localhost"?

Thanks for your time, just an odd thing I spent longer than I'd like to admit to figure out.
